yang/ietf-l2vpn-ntw.yang

```diff
@@ -1045,36 +1045,87 @@ module ietf-l2vpn-ntw {
                           description
                             "p2mp-replication  TBC";
                         }
-                        leaf arp-proxy {
-                          type boolean;
-                          default "false";
-                          description
-                            "Enables (when set to 'true') or disables (when set
-                             to 'false') ARP proxy.";
-                        }
-                        leaf arp-suppression {
-                          if-feature "vpn-common:ipv4";
-                          type boolean;
-                          default "false";
-                          description
-                            "Enables (when set to 'true') or disables (when set
-                             to 'false') ARP suppression.";
-                        }
-                        leaf nd-proxy {
-                          if-feature "vpn-common:ipv6";
-                          type boolean;
-                          default "false";
+                        container arp-proxy {
+                          leaf arp-proxy-enable {
+                            type boolean;
+                            default "false";
+                            description
+                              "Enables (when set to 'true') or disables (when set
+                               to 'false') ARP proxy.";
+                          }
+                          leaf arp-suppression {
+                            if-feature "vpn-common:ipv4";
+                            type boolean;
+                            default "false";
+                            description
+                              "Enables (when set to 'true') or disables (when set
+                               to 'false') ARP suppression.";
+                            reference
+                              "draft-ietf-bess-evpn-proxy-arp-nd-13
```

Comment:
```suggestion
                              "```

Reply:
We can't cite I-Ds. If there is text from that I-d to discuss what arp suppression is about, we can include it in the description